docs/core/integration-quality-scale/rules/iqs043.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "IQS043 - The documentation provides removal instructions"
+---
+
+IQS043 is as follows:
+
+The documentation provides removal instructions
+
+## Reasoning
+
+Removing a device or service from Home Assistant isn't always straightforward.
+The documentation should provide clear instructions on how to remove it, even if it's as simple as "go the integrations page and click delete".
+
+## Example implementation
+
+```markdown
+To remove the integration, go to the integrations page and click delete.
+Make sure that after you delete the integration, you go to the app of the manufacturer and remove the Home Assistant integration from there as well.
+```
```

Comment:
should we also add a note about removing any stored credentials? Or is that a separate topic?

Reply:
Oh, that makes sense for integrations that have application credentials like spotify and withings. I think that is just another example of how it will be implemented. I don't think we necessarily have to add it to the example for now